src/recursion.cpp

```diff
@@ -749,9 +749,11 @@ double class_tree::prior_transition(int s, int t, int level)
     if( s == 1 )   // alternative
     {
         if( t == 1 )  // alternative
-            return( log( 1.0 - eta) + log(gamma) -level*log(beta)  );    
+            // return( log( 1.0 - eta) + log(gamma) -level*log(beta)  );
+            return( log(1.0-eta) + log(beta)  ); 
```

Comment:
Yes can you add that?
On Apr 14, 2016 8:05 PM, "Jacopo Soriano" <notifications@github.com> wrote:

> In src/recursion.cpp
> <https://github.com/jacsor/MRS/pull/7#discussion_r59815556>:
>
> > @@ -749,9 +749,11 @@ double class_tree::prior_transition(int s, int t, int level)
> >      if( s == 1 )   // alternative
> >      {
> >          if( t == 1 )  // alternative
> > -            return( log( 1.0 - eta) + log(gamma) -level*log(beta)  );
> > +            // return( log( 1.0 - eta) + log(gamma) -level*log(beta)  );
> > +            return( log(1.0-eta) + log(beta)  );
>
> I think that the transition matrix should have the structure of the MRS
> paper as a special case. The proposed matrix does not have that structure.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jacsor/MRS/pull/7/files/c272f45ef20631c3ca59d6d989ed9ebdafa878c6#r59815556>
>